client/components/signup-form/index.jsx

```diff
@@ -413,26 +422,26 @@ export default React.createClass( {
 			return this.globalNotice( this.state.notice );
 		}
 		return false;
-	},
+	}
 
 	emailDisableExplanation() {
 		if ( this.props.disableEmailInput && this.props.disableEmailExplanation ) {
 			return (
 				<FormSettingExplanation noValidate={ true }>{ this.props.disableEmailExplanation }</FormSettingExplanation>
 			);
 		}
-	},
+	}
 
 	formFooter() {
 		return (
-			<LoggedOutFormFooter>
+			<LoggedOutFormFooter className={ classNames( { 'is-blended': this.props.isSocialSignupEnabled } ) }>
```

Comment:
I would pass `is-blended` as prop, because we use this class name in LoggedOutFormFooter styles.